pyfakefs/patched_packages.py

```diff
@@ -94,7 +95,14 @@ def __getattr__(self, name):
             return getattr(self._xlrd_module, name)
 
 
-if parsers is not None:
+# From pandas v 1.2 onwards the python fs functions are used even when the engine selected is "c".
+# This means that we don't explicitly have to change the engine.
+patch_pandas = parsers is not None and [int(v) for v in pd.__version__.split(".")] < [
```

Comment:
There are two other places above which use `if parsers is not None`, which also have to be adapted.